hack/build/docker/builder/Dockerfile

```diff
@@ -41,15 +41,11 @@ RUN \
     ( go get -u -d k8s.io/code-generator/cmd/deepcopy-gen && \
     go get -u -d k8s.io/kube-openapi/cmd/openapi-gen ) && \
     ( cd $GOPATH/src/k8s.io/code-generator/cmd/deepcopy-gen && \
-    git checkout kubernetes-1.12.4 && \
+    git checkout kubernetes-1.13.1 && \
```

Comment:
Prob nbd, but should this be `kubernetes-1.13.4`?

Reply:
Changed.